### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by staubina**
**Merge checklist:**
- [x] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [x] `make static` has been run to update webpack bundling if any static content was updated
- [x] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [x] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [x] Translations updated

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
- [ ] Version pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)

### CODE REVIEW

File: enterprise/__init__.py [hunk 1]
```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "2.0.27"
+__version__ = "2.0.28"
```
Comment (crice100) [inline enterprise/__init__.py]: Did we already add release notes for 2.0.28?

Comment (staubina) [reply enterprise/__init__.py]: Yes in a previous PR.

Comment (staubina) [reply enterprise/__init__.py]: https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst#2028---2019-12-3

Comment (staubina) [reply enterprise/__init__.py]: Reference: https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst#2028---2019-12-3

**Pull request merged**